<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testCmakeClass</class>
 <widget class="QMainWindow" name="testCmakeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>testCmake</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnData">
         <property name="text">
          <string>data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAbout">
         <property name="text">
          <string>about</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDataFound">
         <property name="text">
          <string>dataFoutd</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="content" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="testCmake.qrc"/>
 </resources>
 <connections/>
</ui>
